spec/spec_helper.rb

```diff
@@ -1,2 +1,6 @@
 $LOAD_PATH.unshift File.expand_path('../../lib', __FILE__)
 require 'hawk'
+require 'byebug'
```

Comment:
Byebug requires ruby >= 2.0.0. As luck would have it, I'm working in an environment that mandates ruby 1.9.3. Is byebug an absolute must-have?